docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,36 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## November 2024
+
+### Infrastructure and maintenance
+
+- Added [v4.6.13 to v4.6.14 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.6/update_from_4.6/#v4614) which include security fixes.
+- Added [v3.3.40 to v3.3.41 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_3.3/update_from_3.3/#v3341) which include security fixes.
+
+### Content management
+
+- Added [AI Actions documentation](https://doc.ibexa.co/en/master/ai_actions/ai_actions/)
+
+### Search
+
+- Added [new `IsBookmarked` location criterion](https://doc.ibexa.co/en/master/search/criteria_reference/isbookmarked_criterion/)
+- [`IsUserEnabled` is now available on Solr and Elastisearch](https://doc.ibexa.co/en/master/search/criteria_reference/isuserenabled_criterion/)
+
+### Documentation
+
+- When searching from the top bar, you may see, at the bottom of the drop-down suggestion list, a link to a page to see more results.
+
+### PHP API
+
+- Added the following new namespaces, interfaces, and classes to the public PHP API:
```

Comment:
You use "Added (...)" so I think that it's not necessary to add "new" in this sentence. 
It kind of means the same here.

```suggestion
- Added the following namespaces, interfaces, and classes to the public PHP API:
```